CONTRIBUTING.md

```diff
@@ -59,6 +59,19 @@ run local without the need of CI. Have a look at the
 [Testing](docs/getting-started.md#testing)
 section in the [Developer Guide](docs/getting-started.md).
 
+### Signing your commits
+
+All your commits need to be signed in order to get merged. In short the commit message
+needs to have a line looking like this:
+
+```
+Signed-off-by: John Doe <jdoe@example.org>
+```
+
+This can be done by adding [`--signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to your git command line.
```

Comment:
add maybe ? --signoff / -s

Reply:
Hm, I find this a little redundant, as it's mentioned in the docs I linked?